package.json

```diff
@@ -10,12 +10,17 @@
   "homepage": "https://github.com/code-dot-org/dsco_#readme",
   "scripts": {
     "bootstrap": "lerna bootstrap",
+    "build": "NODE_ENV=development lerna run build",
+    "build:prod": "NODE_ENV=production lerna run build",
```

Comment:
the [`lerna run` command](https://github.com/lerna/lerna/tree/main/commands/run#readme) will run the relevant command for every package that contains that script. so here, lerna will look through every `package.json` within `packages/*` and if it has a `build` command, that command will be run. this means that every package's `package.json` needs to have a `build` command, or it will be ignored when `npm run build` is run from the root directory.